chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+Its purpose is to specify the characteristics and linkages of those datasets.
+Some basic characteristics should filled in during the planning stage.
+Example of those characteristics are:
+What is the unit of observation for each dataset?
+What is the name of the ID variable that uniquely and fully identifies each dataset?
+Should any datasets be possible to merge one-to-one?
+And if so, will the have the same unique identifier?
+Should any datasets be possible to merge many-to-one (for example school and student)?
+And if so, what identifier will be used?
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this info,
+will save you a ton of time later on.
+
+Other characteristics should be updated as the project progresses.
+Example of those characteristics are:
+Where will the data be obtain from?
+Where in the project folder are the raw original data stored
+and where are back-ups of the raw data stored?
+See the wiki article linked to above for an template that you can use for your project.
+
+If you have multiple datasets with the same unit of observation
+then you should create a \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+for each of those units of observation and include that in the required datasets.
+The master dataset should include an ID variable
+that can be used to link or merge all datasets with that unit of observation.
+It should also list project wide time-invariant information
+like sampling and treatment status.
+Additionally, any unit of observation
+that will be used in sampling or treatment assignment
+should have a master dataset,
+and that dataset should be used when sampling or assigning treatment.
+
+\section{Taking research design into account when planing your data work}
+
+The most important steps in data work planning
+to correctly translate a research design into data work
+are sampling and treatment assignment.
+Therefore, understanding and correctly implementing sampling and treatment assignment
+is essential to ensuring that data is acquired properly and that research
+is correctly carried out in the field, so that the results
+are valid interpretations of your research question.
+
+Not all projects need to do sampling and treatment assignment.
+Sampling is almost always required when you collect your own data,
+as it is often prohibitively expensive to collect data on all individuals.
+However, if the data you use is, for example,
+the tax filings of all companies in a country,
+then you can estimate the treatment effect
+using the full population and not a sample of it.
+Such big data might need a cluster of computers to process,
+so you might still want to know how make a representative sample,
+so that you can develop your code on a local computer
+before running it on an expensive computer cluster.
+
+Treatment assignment is only applicable if your research design is experimental,
```

Comment:
this section specifically focuses on randomized assignment. it's useful to make the point somewhere that it is always important to have a variable in the dataset identifying treatment (whether or not that is randomized). that probably is better covered in the master dataset section or research design. but useful to note again here, and probably to refer to this as 'randomized treatment assignment'

Reply:
ok. That should probably be brought up the first time here:

https://github.com/worldbank/d4di/blob/1bc431cd5417dcd2fc21e180128a91eff1e450c8/chapters/sampling-randomization-power.tex#L88-L94

But then be mentioned again where we discuss master data sets.